index.js

```diff
@@ -110,23 +110,26 @@ export default class TextMarquee extends PureComponent {
       onMarqueeComplete
     } = this.props
     this.setTimeout(() => {
-      Animated.timing(this.animatedValue, {
-        toValue:         -this.textWidth - repeatSpacer,
-        duration:        duration || children.length * 150,
-        easing:          easing,
-        isInteraction:   isInteraction,
-        useNativeDriver: useNativeDriver
-      }).start(({ finished }) => {
-        if (finished) {
-          if (onMarqueeComplete) {
-            onMarqueeComplete()
-          }
-          if (loop) {
-            this.animatedValue.setValue(0)
-            this.animateScroll()
+      if(!isNaN(-this.textWidth - repeatSpacer)) {
```

Comment:
Just a nitpick, can you pull out `-this.textWidth - repeatSpacer` into its own variable above this since it's repeated on line 115 too.

Reply:
Sorry about that. That was an oversight on my part. It's been updated now.